basic/atm/ATM.py

```diff
@@ -0,0 +1,92 @@
+class ATM:
+    PATH_SAVING_FILE = "ATM_data_base.txt"
+
+    def __init__(self):
+        self._accounts_dict = {}
+        with open(ATM.PATH_SAVING_FILE, "r") as f:
+            account_lst = f.readlines()
+            for account in account_lst:
+                account = account.split()
+                self._accounts_dict[account[0]] = [account[1], int(account[2])]
+        self.main_atm()
+
+    def main_atm(self):
```

Comment:
this function should not be at the ATM class.
the supplies some api, and other function should use it.
In addition, this function is very long, spilt it.
in general, you should see the entire function when you look at the code